mart/attack/adversary.py

```diff
@@ -162,38 +179,53 @@ def forward(
         self,
         input: Union[torch.Tensor, tuple],
         target: Union[torch.Tensor, Dict[str, Any], tuple],
-        model: torch.nn.Module,
-        **kwargs
+        model: Optional[torch.nn.Module] = None,
```

Comment:
Is `model` really optional? Or should this be be a required keyword argument?

Reply:
`model` is a required keyword argument.